package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@Aksh-Bansal-dev please make clear the reason for having 3 package.json files.

Reply:
One for express server, one for react, one for husky(pre-commit linting)